docs/cli/inv-from-cvp.md

```diff
@@ -8,21 +8,34 @@
 
 In large setups, it might be beneficial to construct your inventory based on CloudVision. The `from-cvp` entrypoint of the `get` command enables the user to create an ANTA inventory from CloudVision.
 
+!!! info
+    The current implementation only work towards a local CloudVision instance, not for CVaaS.
+
 ### Command overview
 
 ```bash
-anta get from-cvp --help
 Usage: anta get from-cvp [OPTIONS]
 
-  Build ANTA inventory from Cloudvision
+  Build ANTA inventory from Cloudvision.
+
+  NOTE - CVaaS connection via token is not supported today in the code.
+
+  TODO - handle get_cv_token, get_inventory and get_devices_in_container
+  failures.
```

Comment:
We see this when doing `anta get from-cvp --help`. Since users can see it, I don't think the TODO should be there. Instead of NOTE, should we put WARNING? Also need improvement on the NOTE message, maybe something like:

Authentication to CloudVision as a Service (CVaaS) with a token is not supported.